meadow-client/src/MainFrame.purs

```diff
@@ -473,6 +432,25 @@ evalF (Undo next) = do
         Nothing -> ms
         Just netail -> netail
 
+evalF (HandleBlocklyMessage msg next) = do
+  case msg of
+    Initialized -> pure unit
+    (CurrentCode code) -> do
+      marloweEditorSetValue code (Just 1)
+      assign _view Simulation
+  pure next
+
+evalF (SetBlocklyCode next) = do
+  mSource <- marloweEditorGetValue
+  case mSource of
+    Nothing -> pure unit
+    Just source -> do
+      setBlocklyCode source
+      assign _view BlocklyV
+      _ <- resizeBlockly
```

Comment:
FWIW you could replace this and the `pure unit` with `void resizeBlockly` I believe.

Reply:
fixed